content/docs/examples/advanced-gateways/http-proxy/index.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Connect to an external HTTPS proxy
+description: Describes how to configure Istio to let applications use an external HTTPS proxy.
+weight: 60
+keywords: [traffic-management,egress]
+---
+The [Configure an Egress Gateway](/docs/examples/advanced-gateways/egress-gateway/) showed how you can direct traffic to
+external services from your mesh via an Istio edge component called _Egress Gateway_. However, there are cases when you
+must use an external, legacy (non-Istio) HTTPS proxy to access external services. For example, your company may already
+have such a proxy in place and all the applications within the organization may be required to direct their traffic
+through it. This example shows how to enable access to an external HTTPS proxy.
+
+## Before you begin
+
+*   Setup Istio by following the instructions in the [Installation guide](/docs/setup/).
+
+*   Start the [sleep]({{< github_tree >}}/samples/sleep) sample
+    which will be used as a test source for external calls via the proxy.
+
+    If you have enabled
+    [automatic sidecar injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection), do
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/sleep/sleep.yaml@
+    {{< /text >}}
+
+    otherwise, you have to manually inject the sidecar before deploying the `sleep` application:
+
+    {{< text bash >}}
+    $ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@)
+    {{< /text >}}
+
+    Note that any pod that you can `exec` and `curl` from would do.
+
+*   Create a shell variable to hold the name of the source pod for sending requests to external services.
+    If you used the [sleep]({{<github_tree>}}/samples/sleep) sample, run:
```

Comment:
Not needed, you already established that you are using the sleep sample. It helps to focus on only one flow. In this case, the flow concerning the sleep sample. Avoid referencing other possible flows to avoid confusing your readers.

Reply:
Addressed in a138dba0c850651b88f5af7f2d558486b183c9ca.